scss/forms/_labels.scss

```diff
@@ -21,6 +21,21 @@
   color: $accessible-orange;
   content: "*";
 }
+
+.form-helper {
```

Comment:
(Firefox || Chrome) + macOS, there's an issue with the vertical alignment:

![Screenshot 2022-11-30 at 13 01 09](https://user-images.githubusercontent.com/17381666/204791075-555ee27c-3aa9-4fe9-b801-de77f37e70dc.png)